app/src/main/java/ru/vssemikoz/newsfeed/DAO/NewsItemDAO.java

```diff
@@ -0,0 +1,30 @@
+package ru.vssemikoz.newsfeed.DAO;
+
+import androidx.room.Dao;
+import androidx.room.Delete;
+import androidx.room.Insert;
+import androidx.room.OnConflictStrategy;
+import androidx.room.Query;
+
+import java.util.List;
+
+import ru.vssemikoz.newsfeed.models.NewsItem;
+
+@Dao
+public interface NewsItemDAO {
+
+    @Query("SELECT * FROM NewsItem")
+    List<NewsItem> getAll();
+
+    @Query("SELECT * FROM NewsItem WHERE published_at < :first AND published_at > :second")
+    List<NewsItem> getNewsByDate(String first, String second);
+
+    @Insert
+    void insertAll(List<NewsItem> newsItems);
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    void insertUnique(List<NewsItem> newsItems);
+
+    @Delete
+    void delete(List<NewsItem> newsItem);
```

Comment:
Why have you called this method `delete()` but not `deleteAll()` if you're passing a list there? BTW, there's now need in calling you parameters `newsItems`, `news` also fits well :)